resources/assets/sass/app.scss

```diff
@@ -310,6 +316,9 @@ body, html {
 
 .nav>li>a:focus, .nav>li>a:hover {
   background-color: #00b0f0;
+  -webkit-transition: background-color 0.2s, color 0.2s, -webkit-transform 0.2s;
```

Comment:
This can probably be removed - it's been supported for years now without a vendor prefix